app/routes/components/timer/Timer.js

```diff
@@ -50,7 +37,7 @@ class Timer extends Component {
   render() {
     const { timerStyle } = this.props;
 
-    const time = Timer.msToTime(Math.abs(this.state.elapsedTime));
+    const time = msToTime(Math.abs(this.state.elapsedTime));
```

Comment:
nope! this is pure genius! 
This is could be a countdown or a timer! depending on sign!
Whatever, I just need to convert ms to time (from race start or to race start). 
Applause! 🎉